src/gui/csvImport/CsvImportWidget.cpp

```diff
@@ -254,14 +271,14 @@ QSharedPointer<Database> CsvImportWidget::buildDatabase()
 
         // Icon
         bool ok;
-        int icon = m_parserModel->data(m_parserModel->index(r, 7)).toInt(&ok);
+        int icon = m_parserModel->data(m_parserModel->index(r, 8)).toInt(&ok);
```

Comment:
These magic numbers seem very fragile to me. Can we name or automatically derive them somehow?

Reply:
Yah i agree they are very magical

Reply:
We will fix these later, this is rather static anyway